File: Back-End/C#/01_BOL/DetailsWorkerInProjects.cs
using System.Collections.Generic;

namespace _01_BOL
{
    public class DetailsWorkerInProjects
    {
        public int UserId { get; set; }
        public string TeamLeaderName { get; set; }
        public string Name { get; set; }
        public string Status { get; set; }
        public int Hours { get; set; }
        public List<WorkerHours> ActualHours { get; set; }
    }
}

File: Front-End/Angular/For PHP/src/app/shared/models/details_worker_in_projects.ts
import { WorkerHours } from "../../imports";

export class DetailsWorkerInProjects {
    public  UserId :number;
    public  Name :string;
    public  Status :string;
    public  Hours :number
    public  ActualHours:WorkerHours[]=[];
    public  TeamLeaderName :string;   
}
